UnityProject/Assets/Scripts/Core/Lighting/LightSource.cs

```diff
@@ -305,7 +306,7 @@ private void CheckAudioState()
 					else
 					{
 						SoundManager.ClientPlayAtPositionAttached(ambientSoundWhileOn,
-							gameObject.RegisterTile().WorldPosition, gameObject, loopKey, false, true);
+							gameObject.RegisterTile().WorldPosition, gameObject, loopKey, false, false);
```

Comment:
This will cause a bug where audio will not play until it loops while you're in range. This will make light sounds be abruptly played.

Reply:
well the bool is global or not,  originally the sound was set to play globally and I presume that is a bug

Reply:
its not a bug, sounds like these should stay global so they play correctly when people move to them